wp-content/themes/haruoshi_origin/footer.php

```diff
@@ -8,7 +8,7 @@
 ?>
 	</main>
 	<footer class="site-footer">
-		<p>&copy; haruoshi. 2022.</p>
+		<p class="site-footer__copyright">&copy; 2020 My Work</p>
```

Comment:
いま2022年だしこれでいいんじゃない？笑
```suggestion
		<p class="site-footer__copyright">&copy; 2022 haruoshi.</p>
```

Reply:
たしかに